routes/events.js

```diff
@@ -0,0 +1,65 @@
+var express = require('express');
+var router = express.Router();
+var db = require('../db/db');
+var ObjectId = require('mongodb').ObjectId;
+
+// GET all Events for a Group
+router.get('/:groupId', function(req, res) {
+  db.Event.find({ 'groupId': ObjectId(req.params.groupId) }, function (err, events) {
+    if (err || events === null)
+      res.status(400).send([]);
+    else
+      res.status(200).send(events);
+  });
+});
+
+// POST a new event for a Group
+router.post('/new/:groupId', function(req, res) {
```

Comment:
if you wanna be consistent about route naming, do /:groupId/new, like you did w/ delete